### DISCUSSION TITLE
Accessing entity by String identifiers

### DISCUSSION THREAD

**Discussion by jeteve** [bevyengine/bevy#13585] [Q&A] [answered]
I'm considering using bevy_ecs for a project, and would like to access entities by a String ID (as entities will come from the outside with their String identifiers).

Is there any idiomatic way to do this using bevy or do I need to maintain my own entity dictionary?

**Comment by alice-i-cecile** [comment, marked as answer]
The `Name` component in bevy_core is the standard way to do this :) You'll have to maintain your own dictionary/index though if you want to look up entities by name.

**Reply by jeteve** [reply]
Thanks!
